src/python/CMakeLists.txt

```diff
@@ -28,6 +30,14 @@ set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fno-strict-aliasing")
 set (CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -fno-strict-aliasing")
 set (CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} -fno-strict-aliasing")
 
+IF (NOT SKBUILD)
+    IF (${CMAKE_VERSION} VERSION_LESS "2.8.0")
```

Comment:
Would it be sensible to bump the minimum required version fo CMake ?

Reply:
@Conan-Kudo